src/tms_service.ts

```diff
@@ -45,6 +52,24 @@ type EmsRasterStyle = {
 };
 
 export class TMSService extends AbstractEmsService {
+  /*
+  List of supported languages with labels and OMT code
+  */
+  public static SupportedLanguages = {
+    en: { label: 'English', omtCode: 'en' },
+    'ch-CN': { label: 'Chinese', omtCode: 'zh' },
+    'ja-JP': { label: 'Japanese', omtCode: 'ja' },
+    'fr-FR': { label: 'French', omtCode: 'fr' },
+    es: { label: 'Spanish', omtCode: 'es' },
+    ar: { label: 'Arabic', omtCode: 'ar' },
+    'hi-IN': { label: 'Hindi', omtCode: 'hi' },
+    'ru-RU': { label: 'Russian', omtCode: 'ru' },
+    'pt-PT': { label: 'Portuguese', omtCode: 'pt' },
+    it: { label: 'Italian', omtCode: 'it' },
+    de: { label: 'German', omtCode: 'de' },
+    ko: { label: 'Korean', omtCode: 'ko' },
+  };
```

Comment:
Kibana's `i18n.getLocale()` function uses a normalizer to always return lowercase characters.

```suggestion
    'ja-jp': { label: 'Japanese', omtCode: 'ja' },
    'fr-fr': { label: 'French', omtCode: 'fr' },
    es: { label: 'Spanish', omtCode: 'es' },
    ar: { label: 'Arabic', omtCode: 'ar' },
    'hi-in': { label: 'Hindi', omtCode: 'hi' },
    'ru-ru': { label: 'Russian', omtCode: 'ru' },
    'pt-pt': { label: 'Portuguese', omtCode: 'pt' },
    it: { label: 'Italian', omtCode: 'it' },
    de: { label: 'German', omtCode: 'de' },
    ko: { label: 'Korean', omtCode: 'ko' },
  };
```

Reply:
Fixed with the new array layout.